docs/1.5/extensions/attributes.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+title: Attributes Extension
+description: The AttributesExtension allows HTML attributes to be added from within the document.
+---
+
+# Attributes
+
+The `AttributesExtension` allows HTML attributes to be added from within the document.
+
+## Attribute Syntax
```

Comment:
Missing at the top:

```markdown
## Installation

This extension is bundled with `league/commonmark`. This library can be installed via Composer:

~~~bash
composer require league/commonmark
~~~

See the [installation](/1.5/installation/) section for more details.
```

Reply:
I'll add this after we merge this into `1.5` and then into `master` (since `master` has all of these installation instructions)

Reply:
This has been added: https://commonmark.thephpleague.com/1.5/extensions/attributes/#installation